README.md

```diff
@@ -117,7 +117,7 @@ Usage and Deployment
 --------------------
 
 The application will be written in Java, using Spark 1.2.x+, 
-[Hadoop](http://hadoop.apache.org/) 2.5.x+, [Tomcat](http://tomcat.apache.org/) 8.x+, 
+[Hadoop](http://hadoop.apache.org/) 2.6.x+, [Tomcat](http://tomcat.apache.org/) 8.x+,
```

Comment:
I'd like to leave the requirement at Hadoop 2.5. I actually think it works with earlier versions, but am comfortable only guaranteeing that it will work with 2.5+. The build references 2.6 and that's fine (and maybe at some point we have a separate build profile to check 2.5 compatibility) but I'm really not worried as the use of HDFS APIs is so simple.

For other components, like Spark and Kafka, I think we do need to require the very latest versions.

I wouldn't even mind killing the whole 'Availability' section as the whole thing is ready now in alpha form and there won't be these distinctions going forward.